tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = check-isort, check-flake8, py33, py34, py35, py36
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/pet
+
+deps =
+    -rrequirements.txt
+
+commands =
+    py.test -vv tests.py {posargs:--cov=morizon --cov-report=term-missing}
+
+[testenv:check-isort]
+# isort configurations are located in setup.cfg
+deps = isort==4.2.2
+commands = isort -rc -c morizon
+
+[testenv:run-isort]
+# isort configurations are located in setup.cfg
+deps = isort==4.2.2
+commands = isort -rc morizon
+
+[testenv:check-flake8]
+# flake8 configurations are located in setup.cfg
+deps = flake8==2.5.1
+commands = flake8 morizon
```

Comment:
new line at the end 

https://stackoverflow.com/questions/36043061/automatically-add-newline-on-save-in-pycharm